src/commands/ssh-keys.js

```diff
@@ -0,0 +1,114 @@
+import * as Interact from '../models/interact.js';
+import colors from 'colors/safe.js';
+
+import { Logger } from '../logger.js';
+import { existsSync, readFileSync } from 'node:fs';
+import { executeCommand, openBrowser } from '../models/utils.js';
+import { getKeys, addKey, removeKey } from '../models/ssh-keys.js';
+
+/**
+ * List SSH keys of the current user
+ * @param {object} params The command parameters
+ * @param {string} params.options.format The output format
+ */
+export async function list (params) {
+  const { format } = params.options;
+  const keys = await getKeys();
+
+  switch (format) {
+    case 'json': {
+      Logger.printJson(keys);
+      break;
+    }
+    case 'human':
+    default: {
+      printKeysHuman(keys);
+    }
+  }
+}
+
+/**
+ * Add a SSH key to the current user
+ * @param {{ namedArgs: { "ssh-key-name": string, "ssh-key-path": string } }} params The command parameters
+ */
+export async function add (params) {
+  const { 'ssh-key-name': keyName, 'ssh-key-path': filePath } = params.namedArgs;
+
+  if (!existsSync(filePath)) {
+    Logger.error(`File ${filePath} does not exist`);
+    return;
+  }
+
+  const fileContent = readFileSync(filePath, 'utf8').trim();
+  Logger.debug(`SSH key file content: ${fileContent}`);
+
+  executeCommand(addKey, [keyName, fileContent]);
+}
+
+/** Remove a SSH key from the current user
+ * @param {{ namedArgs: { "ssh-key-name": string } }} params The command parameters
+ */
+export async function remove (params) {
+  const { 'ssh-key-name': keyName } = params.namedArgs;
+  executeCommand(removeKey, [keyName]);
```

Comment:
oh noo! v2 API sucks:
* it fails with an error 500 when trying to delete an unknown ssh-key.
* and there is no GET or HEAD http method available to get a single ssh-key neither

Maybe we should verify if it exists by using an `await getKeys()`